src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -113,6 +119,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     if (request.getParameter(PARAM_TYPE) == null) {
       return Optional.of(ERROR_MISSING_COMMENT_TYPE);
     }
+    if (request.getParameter(PARAM_TRANSCRIPT_LINE) == null) {
+      return Optional.of(ERROR_MISSING_TRANSCRIPT_LINE);
+    }
```

Comment:
Is this an error though? It is possible for a Lecture to have no transcript right?

Reply:
talked to amy about how we want to handle this, and had decided to make this param optional